scribus/plugins/scriptplugin/cmdtext.h

```diff
@@ -197,6 +197,19 @@ May throw ValueError if the font cannot be found.\n\
 /*! Set font features */
 PyObject *scribus_setfontfeatures(PyObject * /*self*/, PyObject* args);
 
+ /*! docstring */
+PyDoc_STRVAR(scribus_setlanguage__doc__,
+QT_TR_NOOP("setlanguage(\"language\", [\"name\"])\n\
+\n\
+Sets the font of the text frame \"name\" to \"language\". If there is some text\n\
```

Comment:
This should be `set the language of…`, but why do we need any changes to python here?